examples/lighting-app/efr32/BUILD.gn

```diff
@@ -52,6 +58,28 @@ show_qr_code = true
 if (efr32_board == "BRD4166A" || efr32_board == "BRD4180A") {
   show_qr_code = false
 }
+if (use_rs911x || use_wf200) {
+  wifi_sdk_dir = "${chip_root}/third_party/efr32_sdk/repo/matter/wifi"
+  if (use_rs911x) {
+    wiseconnect_sdk_root  = "${chip_root}/third_party/wiseconnect-wifi-bt-sdk"
+    import ("${wifi_sdk_dir}/rs911x/rs911x.gni")
+  } else {
+    import ("${wifi_sdk_dir}/wf200/wf200.gni")
+  }
+}
+efr32_lwip_defs = [
+    "LWIP_NETIF_API=1",
+]
+if (use_rs911x || use_wf200) {
+  efr32_lwip_defs += [
+    "LWIP_IPV4=1",
```

Comment:
how about the CHIP_ENABLE_IPV4 setting? can we build rs911x or wf200 that are ipv6 only?

Long term chip goal is to be IPv6 only since thread devices cannot work on ipv4 and we need all chip devices to interoperate.

Reply:
All the WiFi ports (of other hardware ESP32,P6, Linux) all use IPv4 in addition to IPv6. It is not possible to build for IPv6 only